### DISCUSSION TITLE
Is it possible to place AR object to only floor not on desk or something just like this?

### DISCUSSION THREAD

**Discussion by canberka** [google/model-viewer#2492] [General]
What am i trying to is placing the object to only floor.

Example;   

I have a 3d bed model and i want to see this model instead of my real bed but, as you know 3d AR bed will getting on top the real bed.

Is there a way to bypass it? Like ARPlane.classification on ARKit

**Comment by elalish** [comment]
Not yet. That is an interesting use case though.
